bot.js

```diff
@@ -8,6 +8,9 @@ logger.remove(logger.transports.Console, {
 });
 logger.level = 'debug';
 
+//Channel
+var num_Channel = "525249065039036426";
```

Comment:
Do not mix `camelCase` with `underscore_case`. Just rename the variable to `numChannel`. Actually, the comment and the variable name are not descriptive. Means it is difficult to get the meaning of the constant value without reading the whole code. I think it is better to call it something like `ALLOWED_CHANNEL_ID`. Take a look at the Muammar's [bot.js](https://github.com/it-muslim/discord-bots/blob/master/butler-bot/bot.js)
P.S. Uppercase with underscores is usually used for global constants.